.github/workflows/speech-train-data-ci-cd.yml

```diff
@@ -319,28 +316,26 @@ jobs:
       # the variable is a valid GUID with 32 hexadecimal characters.
       - name: Test the new Speech model
         run: |
-          speech test create -n test_from_train_data_update_${{ env.CURRENT_COMMIT_HASH }} -a ${{ env.AUDIO_TRANS_TEST_ID }} -m ${{ env.CUSTOM_SPEECH_MODEL_ID }} -lm ${{ env.CUSTOM_SPEECH_MODEL_ID }} --wait > ${{ env.TEST_BUILD_FOLDER_PATH }}/test-output.txt
-          test_id=$(cat ${{ env.TEST_BUILD_FOLDER_PATH }}/test-output.txt | sed -n '3p')
+          spx csr evaluation create --name test_from_train_data_update_${CURRENT_COMMIT_HASH} --model1 @my.model.url --model2 @my.model.url --dataset @my.testing.datasets --output url @my.test.result --wait 
```

Comment:
Same comment as before where it would be great to delete `--model2 @my.model.url` if possible